embrace-android-sdk/config/detekt/baseline.xml

```diff
@@ -1,6 +1,7 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" ?>
 <SmellBaseline>
-  <ManuallySuppressedIssues/>
+  <ManuallySuppressedIssues></ManuallySuppressedIssues>
   <CurrentIssues>
+    <ID>UseCheckOrError:EmbraceApiService.kt$EmbraceApiService$throw IllegalStateException("Failed to retrieve from Embrace server.")</ID>
```

Comment:
Lets not use the baseline file since they get hidden. Can you suppress with the @Suppress annotation instead?